### ISSUE TITLE
Detect Fn + 1

### ISSUE THREAD

**Issue by cowai** [pop-os/system76-dkms#24]
Hello,
I want to detect and catch keypress Fn + 1 in my application: https://github.com/cowai/pwm-better-fan-control
Is there any way except checking for pwm1 == 255? I also notice that pwm1_enable stays the same when pressing the hotkey. It would be nice for instance pressing fn + 1 made pwm1_enable be 0 or something.

**Comment by jackpot51** [comment]
It should be sending a WMI event, I do not know which one but you could add debugging to find out.

**Comment by cowai** [comment]
@jackpot51 
I can confirm that it does.
0x8F is ON and 0x70 is back to auto.

Would you accept a PR for this to throw a keycode? Do you have any suggestions for what key that should be?

**Comment by jackpot51** [comment]
I do not recommend that a keypress is generated from this driver for those two conditions, as there are no compatible keycodes.

**Comment by cowai** [comment]
@jackpot51 Yes, doesn't feel right. What would you recommend doing instead? Any ideas?

**Comment by cowai** [comment]
This seems interesting: https://lore.kernel.org/patchwork/patch/843457/
